catalog/resource_sql_table_test.go

```diff
@@ -726,6 +726,484 @@ func TestResourceSqlTableUpdateTable_ColumnsTypeThrowsError(t *testing.T) {
 	assert.EqualError(t, err, "changing the 'type' of an existing column is not supported")
 }
 
+func TestResourceSqlTableUpdateTable_ColumnsAdditionAndUpdateThrowsError(t *testing.T) {
```

Comment:
This is 500 lines of test code for 5 tests, but what is actually being tested is around half of that. What can we do to reduce boilerplate in these tests, so they are easier to write/extend in the future and easier to review now? I think there are a lot of opportunities here.